### PR TITLE
Update decora_wifi setup instructions

### PR DESCRIPTION
**Pull request by connorwagner**
## Proposed change

This PR updates the decora_wifi integration to be configurable via the UI
and adds explicit support for Leviton Decora fan controllers.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68070, https://github.com/home-assistant/core/pull/68093
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/decora_wifi.markdown [hunk 1]
```diff
@@ -17,18 +19,11 @@ Supported devices (tested):
 - [DW6HD1-BZ](https://www.leviton.com/en/products/dw6hd-1bz) (Decora Smart Wi-Fi 600W Dimmer)
 - [DW15S-1BZ](https://www.leviton.com/en/products/dw15s-1bz) (Decora Smart Wi-Fi 15A Switch)
 - [DW15P-1BW](https://www.leviton.com/en/products/dw15p-1bw) (Decora Smart Wi-Fi Plug-in Outlet)
+- [DW4SF-1BW](https://www.leviton.com/en/products/dw4sf) (Decora Smart Wi-Fi Fan Speed Controller)
 
-To enable these lights, add the following lines to your `configuration.yaml` file:
+To enable the integration, configure it via the "Add Integration" workflow in the configuration section of the Home Assistant web UI.
```
Comment (frenck) [inline source/_integrations/decora_wifi.markdown]: Use/include our snippet instead:

`{% include integrations/config_flow.md %}`

**Review by frenck — CHANGES_REQUESTED** [review]
Please make sure to add the `ha_config_flow` frontmatter configuration parameter as well.

File: source/_integrations/decora_wifi.markdown (hunk 1 above)
Comment (connorwagner) [reply source/_integrations/decora_wifi.markdown]: Updated, thanks!

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**